src/elements/title/title.visual.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import '../title.js';
+
+describe(`sbb-title`, () => {
+  describeViewports({ viewports: ['zero', 'small', 'medium', 'large', 'ultra'] }, () => {
+    for (const level of ['1', '2', '3', '4', '5', '6']) {
```

Comment:
what about using
```
const cases = {negative: [false, true], level: [1,2,3,4,5,6]}
```
and the describeEach?
With that you can share the template and backgroundColor can be toggled by negative prop.

Reply:
I just assumed we didn't need to test the negative variant with all the levels and wanted to keep a relatively small number of screenshots. But I could live either way, what do you think it's best?

Reply:
excuse me, I misread the closing tag ;) It's fine like you did! :)